source/common/common/backoff_strategy.cc

```diff
@@ -0,0 +1,35 @@
+#include "common/common/backoff_strategy.h"
+
+namespace Envoy {
+
+ExponentialBackOffStrategy::ExponentialBackOffStrategy(const std::uint64_t initial_interval,
+                                                       const std::uint64_t max_interval,
+                                                       const std::uint32_t multiplier)
+    : initial_interval_(initial_interval), max_interval_(max_interval), multiplier_(multiplier),
+      current_interval_(0) {
+  ASSERT(!(multiplier_ <= 0));
```

Comment:
Should multiplier be a `double`? I.e. do we want to force the minimum backoff to be 2x?

Reply:
We could change it to double. If we change it to double, I think minimum backoff can be 1.5 so I will change this like that. Is that fine?